frontend/packages/topology/src/components/page/TopologyView.tsx

```diff
@@ -290,6 +319,10 @@ export const ConnectedTopologyView: React.FC<ComponentProps> = ({
       </Stack>
     </div>
   );
+
+  return _.isFunction(connectDropTarget)
+    ? connectDropTarget(topologyViewComponent)
+    : topologyViewComponent;
```

Comment:
will using typeof make sense here? if yes we can avoid `lodash` import as well then